src/components/base/text/Text.js

```diff
@@ -125,7 +136,7 @@ Container.propTypes = {
     PropTypes.node,
   ]),
   className: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
-  type: PropTypes.oneOf(["text1", "text2", "text3", "text4", "text5", "text6"]),
-  tag: PropTypes.oneOf(["p", "span", "div", "label"]),
+  type: PropTypes.oneOf([...textTypes, ...titleTypes]),
```

Comment:
Det kunne være nice med noget ala  `export const PropTypes_TextTypes = `PropTypes.oneOf([...textTypes, ...titleTypes])` som gør at vi kan bruge denne PropType andre steder, i stedet for bare `PropTypes.string`, f.eks. :thinking:

Reply:
kan eksporteres når den har grund til at blive brugt andre steder. Tænker ikke proptypes typisk bliver passed rundt på den måde

Reply:
Gør den:
`IconButton.js`
`IconLink.js`
`TitleRenderer.js`

Men det kan selvfølgelig godt være at jeg har brugt dem forkert :sweat_smile:

Reply:
Ah okay, forstår lidt bedre. 
Men den bruger Title, så ville den ikke brokke sig når den i sidste ende sender et off tag med? Kender ikke proptypes så dybt, virker den runtime?

Reply:
Den kan vi lige snakke om imorgen